app/models/prisoner.rb

```diff
@@ -5,12 +5,17 @@ class Prisoner
   attribute :number, String
   attribute :prison_name, String
 
-  validates_format_of :number, with: /\A[a-z]\d{4}[a-z]{2}\z/i, message: "must be a valid prisoner number" # eg a1234aa
-  validates_inclusion_of :prison_name, in: Rails.configuration.prison_data.keys, message: "must be chosen"
+  validates_format_of :number,
+    with: /\A[a-z]\d{4}[a-z]{2}\z/i,
+    message: "must be a valid prisoner number" # eg a1234aa
+  validates_inclusion_of :prison_name,
```

Comment:
Not something to handle in this PR worth mentioning.. the preferred rails way for model validations (since RoR 3) is through the use of <a href="http://thelucid.com/2010/01/08/sexy-validation-in-edge-rails-rails-3/">'sexy' validations</a>.